### DISCUSSION TITLE
Configuration Checker warns that 'CertificateProfile' allows ECC but 'keyEncipherment' is enabled, despite 'Forbid encryption usage for ECC keys' enabled

### DISCUSSION THREAD

**Discussion by uedvt359** [Keyfactor/ejbca-ce#521] [Q&A] [answered]
Hi, we have a question regarding the Configuration Checker. It shows the warning, that keyEncipherment' is enabled for an RSA+ECC profile. We thought that checking the `Forbid encryption usage for ECC keys` box would fix the issue, but the warning remains. Are we misunderstanding what this checkbox does?

Version : EJBCA 8.2.0.1 Community

**Comment by svenska-primekey** [comment]
If there is any type of encryption happening I would recommend 2 different cert profiles and not to mix RSA with EC.

**Reply by uedvt359** [reply]
we are looking at [this doc](https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/certificate-profiles-overview/certificate-profile-fields#CertificateProfileFields-KeyUsageKeyUsage):

> **Forbid encryption usage for ECC keys**: This flag enables selecting both RSA and ECSDA key algorithms in the Certificate Profile. When selected _Data Encipherment_ and _Key Encipherment_ key usages will be disabled for ECDSA during certificate creation. 

we thought that this means that we can use one certProfile for both.

**Reply by primetomas** [reply, marked as answer]
The configuration checker is deprecated and will be removed. I think it's simply that the configuration checker does not understant the flag "Forbid encryption usage..."

**Reply by uedvt359** [reply]
Thanks, that answers our question!
